projects/msquic/build.sh

```diff
@@ -33,8 +33,11 @@ $CXX $CXXFLAGS -DCX_PLATFORM_LINUX -DQUIC_TEST_APIS \
     -isystem /src/msquic/submodules/googletest/googletest \
     -c fuzz.cc -o fuzz.o
 
+CMAKECACHE=$(find ./build -name 'CmakeCache.txt'
+LIBMSQUICDIR=$(cmake -LAH $CMAKECACHE | grep QUIC_OUTPUT_DIR | cut -d'=' -f 2)
+
 $CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz.o -o $OUT/fuzz \
-    /src/msquic/artifacts/bin/linux/x64_Debug_openssl/libmsquic.a
+    $LIBMSQUICDIR/libmsquic.a
```

Comment:
I see a couple of `openssl` references below as well. Do those not also need fixes?

Reply:
Agree. @nhorman?

Reply:
yes, I'll address those shortly

Reply:
updated, checking ci now